src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const values = document.querySelectorAll('.population');
+const avarage = document.querySelector('.average-population');
+const total = document.querySelector('.total-population');
+let totl = 0;
+let items = 0;
+
+if (avarage && total) {
+  for (let i = 0; i < values.length; i++) {
+    const value = parseFloat(values[i].textContent);
+
+    if (!isNaN(value)) {
+      totl += value;
+      items++;
+    }
+  }
+
+  const avg = Math.round(totl / items);
```

Comment:
Consider adding a check to ensure 'items' is not zero before performing the division to calculate the average. This will prevent potential division by zero errors.